tasks/resource.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "Apply a single Puppet resource",
+  "input_method": "stdin",
+  "parameters": {
+    "type": {
+      "description": "The type of resource to apply",
+      "type": "String[1]"
+    },
+    "title": {
+      "description": "The title of the resource to apply",
+      "type": "String[1]"
+    },
+    "parameters": {
```

Comment:
In the plans we ship with bolt we expose a `params` parameters instead of a `parameters` parameter. Do we want to be consistent here?

Reply:
That works for me